modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3_{context}"]
+= Release notes for {gitops-title} 1.3
+
+{gitops-title} 1.3 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="support-matrix-1-3_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
+[cols="1,1",options="header"]
+|===
+| Feature | {gitops-title} 1.3.0
+| Argo CD
+| GA
+| Argo CD ApplicationSet
+| TP
+| {gitops-title} Application Manager (kam)
+| TP
+| {gitops-title} Service
```

Comment:
-1 on this :)

I now we've had this in previous release notes, but I would call this "developer console integration" instead of Red Hat OpenShift Gitops Service. Please consult with @siamaksade and @wtam2018 :) 

![image](https://user-images.githubusercontent.com/545280/139772907-0d71e8a4-a2b6-432e-a60b-b416b46524d1.png)

Reply:
We has dropped "Red Hat OpenShift GitOps Service" from this table as this service is an internal service.